gpt_index/langchain_helpers/chain_wrapper.py

```diff
@@ -79,7 +80,10 @@ def _predict(self, prompt: Prompt, **prompt_args: Any) -> str:
         # Note: we don't pass formatted_prompt to llm_chain.predict because
         # langchain does the same formatting under the hood
         full_prompt_args = prompt.get_full_format_args(prompt_args)
-        llm_prediction = llm_chain.predict(**full_prompt_args)
+        llm_prediction = retry_on_exceptions_with_backoff(
```

Comment:
my only ask is could this be a boolean param on LLMPredictor that defaults to True? Since LLMPredictor is user-facing, some people may not want the backoff logic. (also add to docstring too).